content/docs/add-react-to-a-new-app.md

```diff
@@ -0,0 +1,42 @@
+---
+id: add-react-to-a-new-app
+title: Add React to a New Application
+permalink: docs/add-react-to-a-new-app.html
+prev: try-react.html
+next: add-react-to-an-existing-app.html
+---
+
+The easiest way to get started on a new React project is by using a starter kit.
```

Comment:
Related to https://github.com/reactjs/reactjs.org/pull/452/files#r158572397: maybe it's worth pointing people looking for something lightweight to "Try React" from this page. I'd love if we could make it clearer that this is a fully-featured setup that isn't *required* for using React but covers common needs for a *single-page application* including linting, testing, optimizations, etc.

Reply:
👍  Added a note here